UQ-eLIPSE/crucible-components
# story/VETHUB-25

To test:
1. yarn run dev
2. Navigate to http://localhost:5173/
3. Submit an answer to the question.

Verify:
1. Button is only clickable when an option is selected.
2. Once clicked, button style and content should change. Content should be "Next" and the color must be blue.
3. When "Next" button is clicked, question selection must be rebooted and you should be in the initial state.

Run tests:
1. yarn run test
